binding/java/test/ai/picovoice/porcupine/PorcupineTest.java

```diff
@@ -22,17 +22,74 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Arrays;
 
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class PorcupineTest {
 
+    private static final String ENVIRONMENT_NAME;
     private Porcupine porcupine;
     private String accessKey = System.getProperty("pvTestingAccessKey");
 
+    static {
+        ENVIRONMENT_NAME = Utils.getEnvironmentName();
+    }
+
+    private static String append_language(String s, String language) {
```

Comment:
Java doesn't do snake case for methods
```suggestion
    private static String appendLanguage(String s, String language) {
```